graylog2-web-interface/src/components/content-packs/ContentPackEntityConfig.test.jsx

```diff
@@ -14,11 +14,9 @@ describe('<ContentPackEntityConfig />', () => {
     };
     const appliedParameter = [{ configKey: 'descr', paramName: 'descrParam' }];
     const parameter = [{ name: 'descrParam', title: 'A descr Parameter', type: 'string' }];
-    const wrapper = renderer.create(<ContentPackEntityConfig
-      entity={entity}
-      appliedParameter={appliedParameter}
-      parameters={parameter}
-    />);
+    const wrapper = renderer.create(<ContentPackEntityConfig entity={entity}
+                                                             appliedParameter={appliedParameter}
+                                                             parameters={parameter} />);
```

Comment:
Don't recall exactly which eslint flag lines up components like this - but maybe we can discuss its usefulness vs just making sure props are tabbed in once?